docs/src/documentation/03-components/05-input/inputgroup/03-accessibility.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/inputgroup/accessibility/
+---
+
+## Accessibility
+
+The InputGroup component has been designed with accessibility in mind, providing a fieldset structure that groups related form controls with proper semantic markup and screen reader support.
+
+### Accessibility Props
+
+| Name           | Type     | Description                                                                                   |
+| :------------- | :------- | :-------------------------------------------------------------------------------------------- |
+| ariaLabel      | `string` | Provides an accessible label for the fieldset when no visible `label` is provided.            |
+| ariaLabelledby | `string` | References the ID of an element that labels the fieldset.                                     |
+| label          | `string` | Provides a visible legend for the fieldset that also serves as the accessible name.           |
+| required       | `bool`   | When `true`, applies `aria-required="true"` to indicate at least one field must be completed. |
+
+### Automatic Accessibility Features
+
+- The component automatically manages ARIA attributes:
+  - `aria-required="true"` when the `required` prop is set
+  - `aria-describedby` on all child elements when InputGroup has `error` or `help` content and the tooltip is shown
+  - Unique feedback ID generation for error/help content association
+- The component automatically manages semantic structure:
+  - Uses `<fieldset>` element to group related form controls
+  - Wraps `label` prop content in `<legend>` element
+- Child component integration:
+  - InputGroup `error`/`help` takes precedence over individual component messages
+  - Child component labels are visually hidden but converted to `ariaLabel` for screen readers
+  - Child component `ariaDescribedby` values are overwritten by InputGroup's accessibility logic
+
+### Keyboard Navigation
+
+- **Tab/Shift + Tab**: Standard form navigation through child controls
+- Individual keyboard interactions are handled by child components (InputField, Select, etc.)
+
+### Best Practices
+
+- Always provide either `label` or `ariaLabel` to ensure the fieldset has an accessible name
+- Use `ariaLabelledby` when referencing an existing heading that describes the group
+- Avoid setting `ariaDescribedby` on child components as InputGroup will completely override these values
+- Use InputGroup's `error` and `help` props for group-level messages rather than individual component messages
```

Comment:
I like the note in the examples regarding this, maybe we can also mention it here that child error/help messages are ignored?